applications/GeoMechanicsApplication/tests/test_set_multiple_moving_load_process.py

```diff
@@ -646,37 +624,25 @@ def test_SetMultipleMovingLoadsMultipleConditionsConfigurationNegative(self):
 
         """
 
-        #create nodes
-        second_coord = [1.0, 0.0, 0.0]
-        third_coord = [2.0, 0.0, 0.0]
-        self.mp.CreateNewNode(1, 0.0, 0.0, 0.0)
-        self.mp.CreateNewNode(2, second_coord[0],second_coord[1],second_coord[2])
-        self.mp.CreateNewNode(3, third_coord[0], third_coord[1], third_coord[2])
-
-        # create condition
-        self.mp.CreateNewCondition("MovingLoadCondition2D2N", 1, [1, 2], self.mp.GetProperties()[1])
-        self.mp.CreateNewCondition("MovingLoadCondition2D2N", 2, [2, 3], self.mp.GetProperties()[1])
+        self.create_three_nodes_condition()
 
         parameters = self.base_parameters
         parameters.AddVector("configuration", [-0.5])
 
         self.mp.ProcessInfo.SetValue(KratosMultiphysics.TIME,
```

Comment:
Same here, could you please move this to a single line to make it consistent with the other changes?